sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaRulesDefinition.java

```diff
@@ -190,12 +202,18 @@ private static String readResource(URL resource) {
     String type;
     String[] tags;
     String defaultSeverity;
+    SecurityStandards securityStandards = new SecurityStandards();
```

Comment:
do we need this initializer?

Reply:
yes, otherwise if the field is not present it will fail on NPE